lib/pliny/sidekiq/middleware/server/log.rb

```diff
@@ -1,8 +1,7 @@
 module Pliny::Sidekiq::Middleware
   module Server
     class Log
-      def initialize(metric_prefix: nil)
-        @metric_prefix = metric_prefix
+      def initialize(opts={})
```

Comment:
Hmm, actually, I'm not sure it's required by the Sidekiq API. The :boom: may have been from internal calls. Looking...